packages/quantic/force-app/main/default/lwc/quanticSort/e2e/fixture.ts

```diff
@@ -0,0 +1,64 @@
+import {SortObject} from './pageObject';
+import {quanticBase} from '../../../../../../playwright/fixtures/baseFixture';
+import {SearchObject} from '../../../../../../playwright/page-object/searchObject';
+import {
+  searchRequestRegex,
+  insightSearchRequestRegex,
+} from '../../../../../../playwright/utils/requests';
+import {InsightSetupObject} from '../../../../../../playwright/page-object/insightSetupObject';
+import {useCaseEnum} from '../../../../../../playwright/utils/useCase';
+
+const pageUrl = 's/quantic-sort';
```

Comment:
Personally I would rename this


```suggestion
const sortUrl = 's/quantic-sort';
```